### ISSUE TITLE
Invalid JSON payload received. Unknown name "tokens" at \'message\': Cannot find field

### ISSUE THREAD

**Issue by yswtrue** [healthjoy/async-firebase#23]
```
        client = cls.get_client()
        message = client.build_android_config(
            data=data,
            title="WOMO",
            body=_(notification),
            priority="high",
            click_action="FLUTTER_NOTIFICATION_CLICK",
        )
        result = await client.push_multicast(device_tokens=tokens, android=message)

```
get this result
```
{'error': {'code': 400, 'message': 'Invalid JSON payload received. Unknown name "tokens" at \'message\': Cannot find field.', 'status': 'INVALID_ARGUMENT', 'details': [{'@type': 'type.googleapis.com/google.rpc.BadRequest', 'fieldViolations': [{'field': 'message', 'description': 'Invalid JSON payload received. Unknown name "tokens" at \'message\': Cannot find field.'}]}]}}
```

**Comment by akalex** [comment]
@yswtrue JFYI, here is the [fix](https://github.com/healthjoy/async-firebase/pull/24)

**Comment by akalex** [comment]
The issue has been fixed in version [1.9.1](https://github.com/healthjoy/async-firebase/releases/tag/v1.9.1)
